release/env_cpu.yml

```diff
@@ -9,9 +9,10 @@ dependencies:
   - pip
   - python
   - requests
-  - tensorflow=2.1.0
   - pip:
-      - opencv-python==4.2.0.*
+      - opencv-python==4.1.0.*
       - pyqt5
+      - opencv-contrib-python-headless
+      - tensorflow==2.1.0
 prefix: ./env/stefann-cpu
 
```

Comment:
Try this:
```
name: stefann-osx
channels:
  - defaults
dependencies:
  - colorama
  - keras
  - numpy
  - pillow
  - pip
  - python
  - requests
  - pip:
      - opencv-contrib-python==4.2.0.*
      - pyqt5
      - tensorflow==2.1.0
prefix: ./env/stefann-osx
```

Check [this comment](https://github.com/prasunroy/stefann/pull/1#issuecomment-623489938) for details.

Reply:
Thank you. It resolved all errors for me and now it works perfectly fine. In my case i used
- opencv-contrib-python-headless.